app/(golems)/golems/skills/[name]/EvalDashboard.tsx

```
@@ -0,0 +1,655 @@
+  );
+}
+
+function PassFailCell({ passed }: { passed: boolean }) {
+  return (
+    <td className="px-3 py-2.5 text-center">
+      {passed ? (
+        <span
+          className="inline-flex h-6 w-6 items-center justify-center rounded-full bg-[#28c84015] text-[#28c840]"
+          aria-label="Passed"
+          title="Passed"
+        >
```

Biome (2.4.6)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)